apidoc/Titanium/Blob.yml

```diff
@@ -90,7 +90,8 @@ properties:
     description: |
         If this blob represents an image, this is the total number of pixels in the image.
         Otherwise it returns the number of bytes in the binary data.
-    platforms: [iphone, ipad]
+    platforms: [android, iphone, ipad]
+    since: {android: "7.2.0"}
```

Comment:
Can the "0.9.0" for iOS be omitted these?

Reply:
There's a default value if unspecified. It's defined in the doc generator, and I think it's actually 0.8.0. I assumed you could omit and it would use the default - but maybe the doc generator needs to be tweaked to allow that?